ocaml/idl/datamodel.ml

```diff
@@ -7107,6 +7107,17 @@ let pool_remove_from_guest_agent_config = call
     ~allowed_roles:_R_POOL_ADMIN
     ()
 
+let pool_set_default_sr = call
+    ~name:"set_default_SR"
+    ~in_product_since:rel_ely
+    ~doc:"Set the default SR for VDIs."
+    ~params:
+      [ Ref _pool, "self", "The pool"
+      ; Ref _sr, "sr", "SR to hold VDIs."
```

Comment:
Calling this `sr` instead of `value` is a change to the API, so clients will no longer work.
Let's keep it as `value`.